doc/source/callbacks.rst

```diff
@@ -909,119 +909,64 @@ a separate process.
 
 In the main process, where the RunEngine is executing the plan, a ``Publisher``
 is created. It subscribes to the RunEngine. It serializes the documents it
-receives and it sends them over a socket to a 0MQ "forwarder device," which
-rebroadcasts the documents to any number of other processes or machines on the
-network.
+receives and it sends them over a socket to a 0MQ proxy which rebroadcasts the
+documents to any number of other processes or machines on the network.
 
 These other processes or machines set up a ``RemoteDispatcher`` which connects
-to the "forwarder device," receives the documents, and then runs callbacks just
-as they would be run if they were in the local ``RunEngine`` process.
+to the proxy receives the documents, and then runs callbacks just as they would
+be run if they were in the local ``RunEngine`` process.
 
 Multiple Publishers (each with its own RunEngine) can send documents to the
-same forwarder device. RemoteDispatchers can filter the document stream based
-on host, process ID, and/or ``id(RunEngine)``.
+same proxy. RemoteDispatchers can filter the document stream based on host,
+process ID, and/or ``id(RE)`` with ``RE`` is a particular instance of
+``RunEngine``.
 
 Minimal Example
 +++++++++++++++
 
-Look for a forwarder device configuration file at
-``/etc/zmq_forwarder_device.yml`` or
-``~/.config/zmq_forwarder_device/connection.yml``. If there isn't one, create
-one:
+Start a 0MQ proxy using the CLI packaged with bluesky. It requires two ports as
+arguments.
 
-.. code-block:: yaml
+.. code-block:: bash
 
-    #~/.config/zmq_forwarder_device.yml
-    {'frontend_port': 5577
-    'backend_port': 5578
-    'host': 'localhost'}  # optional
+    bluesky-0MQ-proxy 5577 5578
 
-In production (e.g., at NSLS-II beamlines) the forwarder device should be
-running in the background as a service. Here is how to start one just for play:
+Alternatively, you can start the proxy using a Python API:
 
-.. code-block:: bash
+.. code-block:: python
 
-    # uses config in /etc/zmq_forwarder_device.yml
-    #  or ~/.config/zmq_forwarder_device/connection.yml
-    $ python bluesky/examples/forwarder_device.py
+    from bluesky.callbacks.zmq import Proxy
```

Comment:
This is a pretty compelling reason not use the `async def run` pattern, but if someone gets here (rather than using the proxy cli tool above they are down the rabbit hole :dragon: .